# geocode_place_get_street_address (place) == "North Haugh π") failed:
# ("\317\200 North Haugh" == "North Haugh \317\200")
# in the json file, house number is π, street name is North Haugh

--- a/geocode-glib/tests/geocode-glib.c	2024-01-31 10:39:25.179480656 -0800
+++ b/geocode-glib/tests/geocode-glib.c	2024-01-31 10:39:51.340038482 -0800
@@ -440,7 +440,7 @@
 
 	g_assert_cmpint (g_list_length (res), ==, 1);
 	place = res->data;
-	g_assert_cmpstr (geocode_place_get_street_address (place), ==, "North Haugh π");
+	g_assert_cmpstr (geocode_place_get_street_address (place), ==, "π North Haugh");
 	g_object_unref (place);
 	g_list_free (res);
 
